pr2eus_moveit/euslisp/pr2eus-moveit.l

```diff
@@ -1,4 +1,8 @@
+(require :robot-interface "package://pr2eus/robot-interface.l")
 (require :robot-moveit "package://pr2eus_moveit/euslisp/robot-moveit.l")
+(require :pr2 "package://pr2eus/pr2.l")
+(require :pr2-utils "package://pr2eus/pr2-utils.l")
```

Comment:
I assume `:robot-interface`, `:pr2` and `:pr2-utils` are dependencies of `pr2-interface`.
Do we have to write them explicitly?
Writing them explicitly is not a common way, I think.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus_openrave/pr2eus-openrave.l#L5